modules/dag-shared/src/main/scala/org/tessellation/dag/transaction/filter/Consecutive.scala

```diff
@@ -0,0 +1,28 @@
+package org.tessellation.dag.transaction.filter
+
+import scala.annotation.tailrec
+
+import org.tessellation.schema.transaction.{Transaction, TransactionReference}
+import org.tessellation.security.signature.Signed
+
+object Consecutive {
```

Comment:
can you create a jira ticket to write a unit test for that?

Reply:
Yep!